src/utils/EnvironmentVariables.ts

```diff
@@ -0,0 +1,17 @@
+import { Logger } from "@aws-lambda-powertools/logger";
```

Comment:
There is a test file for this that can be copied over

Reply:
Done.